docs/release-notes/.FSharp.Core/8.0.400.md

```diff
@@ -6,3 +6,7 @@
 
 * Cache delegate in query extensions. ([PR #17130](https://github.com/dotnet/fsharp/pull/17130))
 * Update `AllowNullLiteralAttribute` to also use `AttributeTargets.Interface` ([PR #17173](https://github.com/dotnet/fsharp/pull/17173))
+
+### Breaking Changes
+
+* Fixed argument exception throwing inconsistency ([#17328](https://github.com/dotnet/fsharp/pull/17328))
```

Comment:
I would maybe mention that it has to do with `ArgumentException` and `ArgumentOutOfRangeException`.
like
```suggestion
* Fixed argument exception throwing inconsistency - accessing an out-of-bounds list index will now throw `ArgumentOutOfRangeException` ([#17328](https://github.com/dotnet/fsharp/pull/17328))
```
Or something like that?

Reply:
Good point, clarified.